### DISCUSSION TITLE
v0.10.3

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#647] [Announcements]
<details>
<summary>中文版</summary>

1. 添加 Anime4K_Restore_Soft_UL、Anime4K_Restore_Soft_VL 和 Anime4K_Restore_UL #605 #602（感谢 @kato-megumi）
2. 支持葡萄牙语（巴西）#628（感谢 @Zoommod）
3. 优化了主窗口样式 #624 #599
4. 添加一个选项允许缩放最大化或全屏的窗口 #639 #559
5. 允许更改用于启动程序的可执行文件 #641 #609
6. 游戏内叠加层支持本地化 #600
7. 优化软件体积 #592 #601 #623

</details>

1. Add Anime4K_Restore_Soft_UL, Anime4K_Restore_Soft_VL and Anime4K_Restore_UL #605 #602 (thanks @kato-megumi)
2. Support Portuguese (Brazil) #628 (thanks @Zoommod)
3. Optimize the style of the main window #624 #599
4. Add an option to allow scaling maximized or fullscreen windows #639 #559
5. Allow changing the executable file used to launch the program #641 #609
6. In-game overlay supports localization now #600
7. Optimize the size of the software #592 #601 #623

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.10.3'>v0.10.3</a>.</em>

**Comment by kakarisu** [comment]
用截圖比較也看不出來Restore的UL跟VL有什麼差別

**Comment by kylestanley28** [comment]
Hi, ive been using your app for about a year now, upscaling games, watching anime etc. just today i decided to check if there is a new version and yep there is, but im getting a jangmin hacktool.inject.csr on virus total on your version 10+, is it a false positive? 

also is it worth upgrading from version 9.03 to version 10.3 if im just watching anime? is there more improvements than anime 4k gan restore vl? i really love your app and im really grateful for it.

**Reply by Blinue** [reply]
Hello, I’m happy to see this program is helpful to you. Some antivirus software treats any unsigned software as a threat, even [the Go language suffers a lot from this](https://go.dev/doc/faq#virus). Magpie is FOSS, you can use it with confidence.

> also is it worth upgrading from version 9.03 to version 10.3 if im just watching anime? is there more improvements than anime 4k gan restore vl? i really love your app and im really grateful for it.

v0.10 added many useful features, but the core scaling function did not change significantly, so it’s okay not to upgrade. My suggestion is to always upgrade to the latest version to get new features and fixes.

**Reply by kylestanley28** [reply]
I used it but i prefer the old one, i did try the upscaling effects but the old one is better probably because im not sure what effects should i add, for now im contented with the one, thank you so much for this app :) its really useful for me.

**Comment by b9348** [comment]


**Reply by Blinue** [reply]
检查缩放模式有没有配置正确
